django_feature_flip/tests.py

```diff
@@ -34,13 +34,55 @@ def test_singleton(self):
         self.assertEqual(id(self.feature_flip), id(self.feature_flip))
 
     def test_enabled_for_not_totally_enabled_feature(self):
-        FeatureFactory(name='my_feature', totally_enabled=False)
+        FeatureFactory(name='my_enabled_feature')
         actor = self._actor(1)
 
-        self.assertFalse(self.feature_flip.enabled('my_feature', actor))
+        self.assertFalse(self.feature_flip.enabled('my_enabled_feature', actor))
 
     def test_enabled_for_totally_enabled_feature(self):
-        FeatureFactory(name='my_feature', totally_enabled=True)
+        FeatureFactory(name='my_disabled_feature', totally_enabled=True)
+        actor = self._actor(1)
+
+        self.assertTrue(self.feature_flip.enabled('my_disabled_feature', actor))
+
+    def test_enabled_for_non_existent_feature(self):
         actor = self._actor(1)
+        try:
```

Comment:
Why do you add a `try` block in test? What is the purpose?

Reply:
This is not the way to asssert for exceptions, please check this: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises